src/config/feature-flags.ts

```diff
@@ -75,9 +76,11 @@ const getLaunchDarklyValue = async <T = boolean | string | number>(flag: Boolean
 };
 
 // Include jiraHost for any FF that needs to be rolled out in stages
-export const booleanFlag = async (flag: BooleanFlags, key?: string): Promise<boolean> =>
+export const booleanFlag = async (flag: BooleanFlags, key?: string): Promise<boolean> => {
+	if (flag === BooleanFlags.USE_NEW_5KU_SPA_EXPERIENCE) return true;
```

Comment:
Why are we returning true for this flag check? Can't we just set it from Launchdarkly?

Reply:
good catch!